apps/modernization-ui/src/pages/patientProfile/Demographics.tsx

```diff
@@ -497,14 +574,46 @@ export const Demographics = ({ patientProfileData, handleFormSubmission, ethnici
 
             <div className="margin-top-6 margin-bottom-2 flex-row common-card">
                 <TableComponent
+                    handleAction={(type, data) => {
```

Comment:
This looks like common code, and functionality for a table component.  Can this be reused as a component?  Even if it is just for this page?

Reply:
We are already getting this prop from the table component as we are using different pop ups for each table so we have to pass the modal ref, and the model ref should not be re usable for each modal that were creating